app/economy/management/commands/get_prices.py

```diff
@@ -111,13 +112,47 @@ def polo():
         print('refreshed {}'.format(b.pk))
         try:
             b._val_usd_db = b.value_in_usdt
+            b._val_usd_db_now = b.value_in_usdt_now
             b.save()
         except Exception as e:
             print(e)
             b._val_usd_db = 0
+            b._val_usd_db_now = 0
             b.save()
 
 
+def cryptocompare():
+    """Handle pulling market data from CryptoCompare.
+       Updates ConversionRates only if data not available."""
+
+    to_currency = 'USDT'
```

Comment:
for Bounties, we only need to iterate through `Bounty.objects.filter(current_bounty=True).all()`